ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/CommandDeduplicationIT.scala

```diff
@@ -620,8 +614,13 @@ final class CommandDeduplicationIT(
           .getTimeModel()
           .flatMap(response => {
             ledger.delayMechanism.delayBy(
-              response.getTimeModel.getMaxSkew.asScala +
-                2 * response.getTimeModel.getMinSkew.asScala
+              Durations.scaleDuration(
+                Durations.asFiniteDuration(
+                  2 * (response.getTimeModel.getMaxSkew.asScala +
+                    response.getTimeModel.getMinSkew.asScala)
```

Comment:
Is this empirically chosen or is there a specific reason?

Reply:
It's chosen because we use the time skews twice, once when calculating if it's a duplicate by using min/max record time and the second time when doing the conversion from offset to duration to account for any possible skew.

Reply:
Could you add a comment about that?

Reply:
There is a comment a few lines above, do you think we should move that one or is it not clear enough?

Reply:
I think adding to it something like "thus, we delay by twice the skews below" would make it clearer.

Reply:
Done